### ISSUE TITLE
Add required `aria-controls` and `aria-expanded` attributes to `c-select` input

### ISSUE THREAD

**Issue by kelseymcmahon** [IntelliTect/Coalesce#652]
In the DOM, the `c-select` does not have all the required attributes for accessibility. It is missing the `aria-controls` and `aria-expanded` and is breaking the WCAG 2.1 level A [guidelines ](https://dequeuniversity.com/rules/axe/4.11/aria-required-attr?application=AxeChrome). When replacing the `c-select` with the Vuetify equivalent in the code, the accessibility error is no longer an issue. 

Here is the difference between the two elements in the DOM:

`c-select`:

`<input size="1" role="combobox" type="text" id="input-v-66" closable-chips="" chips="" no-data-text="No cases available for invite" value="">`

`v-autocomplete`:

<input size="1" role="combobox" type="text" id="input-v-71" aria-expanded="false" aria-controls="menu-v-69" value="">
